### PR TITLE
Pylexibank2

### PR DESCRIPTION
**Pull request by tresoldi**
The dataset is still missing a concept list (it uses `etc/concepts.tsv`), which requires issues https://github.com/lexibank/tls/issues/8 and https://github.com/concepticon/concepticon-data/issues/665

### CODE REVIEW

**Review by LinguList — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ Any dataset specific notes on lexibank decisions/mapping choices etc go in here.
...
 ![Source: 100%](https://img.shields.io/badge/Source-100%25-brightgreen.svg "Source: 100%")
 
-- **Varieties:** 128
-- **Concepts:** 1,523
-- **Lexemes:** 157,589
-- **Synonymy:** 1.36
\ No newline at end of file
+- **Varieties:** 127
+- **Concepts:** 1,581
+- **Lexemes:** 130,556
```
Comment (chrzyki) [inline README.md]: Is the decrease of lexemes accounted for?

Comment (chrzyki) [reply README.md]: There is also a number of problematic splits, e.g.:

- Tuveta-962_oilfromplants-1 `ngayo 1a/2a` `ngayo 1a`
- Tuveta-962_oilfromplants-2 `ngayo 1a/2a` `2a`

And other numbers in the forms:

- Tuveta-183_beard-1 `nyami 9`

Sorry if this was discussed elsewhere.

**Comment by tresoldi** [comment]
@chrzyki Yes, I removed Gweno1, as per #6 , sorry.

I'm checking the split now.

**Comment by tresoldi** [comment]
@chrzyki thanks's for catching that, I forgot to merge the formspec! I'll fix it now.

**Comment by tresoldi** [comment]
I've fixed the cross-reference issues, but I had to use manual approaches that would better fit in FormSpec. I can remove them if desired, but the only other way to solve it is to perform a huge number of `replacements`.

In detail, I am:

- skipping over cross-references with no linguistic material (entries starting with `"see "`)
- using a regular expression to strip references at the end of strings (I know regex are almost always a no-go, but this is short and I commented it in verbose mode).

I am opening an issue on `pylexibank` for this as well. @chrzyki , please check; @LinguList , please re-review; pinging @xrotwang and @SimonGreenhill

**Comment by LinguList** [comment]
Look, this is the typical case, where I would say: please use `lexemes.tsv`. I just counted all "see" entries, and there are less than 30. So why not just pull them out and place them in lexemes, and have them be replaced by "-", and then use "-" as the empty string?

I think, even if it takes less than 100 or so entries, the same should also be done with the numbers. 

Why so? Because this is explicit, and you have no guarantee if

1. your regex captures too many instances, or
2. you are trying to kill birds with cannon balls

The good thing about the form-spec following protocol is here that you can have ALL cases in a uniformly replicable way. The cost is to have some more items in lexemes.tsv, but everybody knows where to find them.

As a general rule for all lexibankers (@SimonGreenhill, @chrzyki, @xrotwang and @LinguList), I would say: try to get as far as you can with formspec only. If you can't, give up the dataset and only report the values (as in `kitchensemitic`).

Again, the rule is, what Sunzi says in the "Art of War": Know when to fight, and know when NOT to fight.

**Comment by tresoldi** [comment]
Ok, moved to `lexemes.csv`. Please check.

**Comment by LinguList** [comment]
could you also push lexemes?

**Comment by tresoldi** [comment]
Sorry @LinguList , what do you mean?

**Comment by tresoldi** [comment]
The `lexemes.csv` file is there https://github.com/lexibank/tls/blob/pylexibank2/etc/lexemes.csv

**Comment by LinguList** [comment]
Okay thanks, saw it now. It did not show up in my summary. So we have less than 500 entries that are "bad", with some 120 000 entires in total, I think that's less than a percent, and can be handled like this. Thanks. Feel free to merge.

**Comment by LinguList** [comment]
Ah, last thing: if you want to be consistent, as we discussed yesterday, just use the `x.id` instead of the `x.number`, as we discussed over satterthwaite-phillips, so everybody has a similar way of making their parameter ids, and it is not guaranteed that the number is unique in a concept list, while the ID is always.

**Comment by tresoldi** [comment]
Oh, right, my bad on the `id`.

**Pull request merged**